### PR TITLE
Update variables_file.rst

### PR DESCRIPTION
**Pull request by nullHawk**
Added _filesize variable as mentioned in "https://github.com/metabrainz/picard/pull/2361"

### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Added _filesize variable as mentioned in "https://github.com/metabrainz/picard/pull/2361"

### CODE REVIEW

File: variables/variables_file.rst [hunk 1]
```diff
@@ -57,3 +57,6 @@ These variables are populated from information found in the audio files themselv
 **_sample_rate**
 
     Number of digitizing samples per second (Hz).
+**_filesize**
+
+   Size of the file in Bytes.
```
Comment (zas) [inline variables/variables_file.rst:62]:
```suggestion
   Size of the file in bytes.
```

**Review by zas — CHANGES_REQUESTED** [review]
AFAIK, units should not be capitalized but if they are named after a proper name (like Celsius). Even if abbreviation is (B for byte, but that's because b is used for bit).

File: variables/variables_file.rst [hunk 2]
```diff
@@ -57,3 +57,6 @@ These variables are populated from information found in the audio files themselv
 **_sample_rate**
 
     Number of digitizing samples per second (Hz).
+**_filesize**
```
Comment (phw) [inline variables/variables_file.rst:60]: Please add a blank line before the `**_filesize**`

**Review by phw — CHANGES_REQUESTED** [review]

**Comment by rdswift** [comment]
@nullHawk, please also add your name to the list of contributors in `about_picard/acknowledgements.rst`.  Note that the ordering is alphabetical by surname.

**Comment by nullHawk** [comment]
> AFAIK, units should not be capitalized but if they are named after a proper name (like Celsius). Even if abbreviation is (B for byte, but that's because b is used for bit).

Thank you for mentioning it, I will keep that in mind from now onwards

**Comment by nullHawk** [comment]
> @nullHawk, please also add your name to the list of contributors in `about_picard/acknowledgements.rst`. Note that the ordering is alphabetical by surname.

Ok, got it

**Review by zas — APPROVED** [review]

**Review by phw — APPROVED** [review]

**Comment by phw** [comment]
Thanks again, merged

**Pull request merged**